phpBB/phpbb/template/twig/tokenparser/event.php

```diff
@@ -18,14 +18,38 @@ class event extends \Twig\TokenParser\AbstractTokenParser
 	/** @var \phpbb\template\twig\environment */
 	protected $environment;
 
+	/** @var \phpbb\event\dispatcher_interface */
+	protected $phpbb_dispatcher;
+
+	/** @var array */
+	protected $template_event_priority_array;
+
 	/**
 	* Constructor
 	*
 	* @param \phpbb\template\twig\environment $environment
 	*/
-	public function __construct(\phpbb\template\twig\environment $environment)
+	public function __construct(\phpbb\template\twig\environment $environment, \phpbb\event\dispatcher_interface $phpbb_dispatcher = null)
 	{
 		$this->environment = $environment;
+		$this->phpbb_dispatcher = $phpbb_dispatcher;
+
+		$template_event_priority_array = [];
+
+		/**
+		* Allow assigning priority to template events
+		*
+		* @event core.twig_tokenparser_constructor
+		* @var	array	template_event_priority_array	Array with template event priority assignments per extension namespace
+		* @since 3.3.12-RC1
+		*/
+		if ($this->phpbb_dispatcher)
+		{
+			$vars = array('template_event_priority_array');
+			extract($this->phpbb_dispatcher->trigger_event('core.twig_tokenparser_constructor', compact($vars)));
+		}
+
+		$this->template_event_priority_array = $template_event_priority_array;
```

Comment:
It probably makes sense to `unset($template_event_priority_array);` as it's not used anymore.